FizzBuzz/FizzBuzz.cs

```diff
@@ -7,26 +7,38 @@ class Program
         //Main Code
         static void Main(string[] args)
         {
-            Console.WriteLine("Please choose enter 1 or 2 for the loop method:" +
+Console.WriteLine("Please choose enter 1 or 2 for the loop method:" +
             "(1) 'For Loop'" +
             "(2) 'While Loop'");
             int Loop_Type = 0;
 
             //Read Loop Type Input
-            Loop_Type = (Convert.ToInt32(Console.ReadLine()));
-
+            try
+            {
+                Loop_Type = (Convert.ToInt32(Console.ReadLine()));
```

Comment:
A different approach is to throw an exception if the value entered is not 1 or 2, instead of checking, since you are wrapping the user input in a try/catch anyways